packages/hash/datastore/postgres/scripts/run-schema-migrations.ts

```diff
@@ -19,7 +18,7 @@ HASH_PG_PASSWORD, if set, will be used as the password, otherwise you
 will be prompted for the password.`;
 
 const main = async () => {
-  const argv = yargs(process.argv.slice(2))
+  const argv = await yargs(process.argv.slice(2))
```

Comment:
We had `yargs` from some sub-dependency. It got upgraded to so the typings got wrong. I added a missing `await` keyword and also added `"@types/yargs"` to `package.json` to avoid similar surprises in the future.